snakemake_executor_plugin_slurm/__init__.py

```
@@ -743,35 +732,32 @@ def test_account(self, account):
         """
         tests whether the given account is registered, raises an error, if not
         """
-        # first we need to test with sacctmgr because sshare might not
-        # work in a multicluster environment
-        cmd = f'sacctmgr -n -s list user "{os.environ["USER"]}" format=account%256'
-        sacctmgr_report = sshare_report = ""
+        cmd = "sshare -U --format Account%256 --noheader"
         try:
             accounts = subprocess.check_output(
```

Ruff (0.12.2)

`subprocess` call with `shell=True` identified, security issue

(S602)